lib/stripe/subscriptions/invoice.ex

```diff
@@ -76,23 +78,32 @@ defmodule Stripe.Invoice do
           footer: String.t() | nil
         }
 
+  @type status_transitions ::
+          list(%{
+            finalized_at: Stripe.timestamp() | nil,
+            marked_uncollectible_at: Stripe.timestamp() | nil,
+            paid_at: Stripe.timestamp() | nil,
+            voided_at: Stripe.timestamp() | nil
+          })
+
   defstruct [
```

Comment:
A lot of fields missing here: ```[warn]  Extra keys were received but ignored when converting Stripe.Invoice: [:account_country, :account_name, :customer_address, :customer_email, :customer_name, :customer_phone, :customer_shipping, :customer_tax_exempt, :customer_tax_ids, :default_tax_rates, :payment_intent, :post_payment_credit_notes_amount, :pre_payment_credit_notes_amount, :total_tax_amounts]```

But can be left for another PR

Reply:
Fixed!  How did you generate these warning messages.  Not `Logger.warn`?

Reply:
Changed this line https://github.com/code-corps/stripity_stripe/blob/master/lib/stripe/converter.ex#L143 from `Logger.debug()` to `Logger.warn()` and run the tests.

When you do, it'll show a lot more warnings but those were unrelated to this PR.